tests/std/tests/P0896R4_views_filter/test.cpp

```diff
@@ -380,4 +380,26 @@ int main() {
 
     STATIC_ASSERT((instantiation_test(), true));
     instantiation_test();
+
+    { // Validate **non-standard guarantee** that predicates are moved into the range adaptor closure, and into the view
+      // object from an rvalue closure
+        struct Fn {
+            Fn()     = default;
+            Fn(Fn&&) = default;
+            Fn(const Fn&) {
+                assert(false);
+            }
+            Fn& operator=(Fn&&) = default;
+            Fn& operator        =(const Fn&) {

```

Comment:
clang-format is being a bad kitty here. I recommend adding a newline to prevent it from attempting to align the `=`.